File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/NoCommand.java
package com.example.try_gameengine.remotecontroller;

import android.view.MotionEvent;

import com.example.try_gameengine.remotecontroller.RemoteController.CommandType;

/**
 * This no command is a default for remote controller.
 * @author irons
 *
 */
public class NoCommand implements Command{
	private int pointerId = -1;
	
	@Override
	public CommandType execute() {
		return CommandType.None;
	}

	@Override
	public boolean checkExecute(float x, float y, MotionEvent event) {
		return false;
	}

	@Override
	public void setMotionEventPointerId(int pointerId) {
		this.pointerId = pointerId;
	}

	@Override
	public int getMotionEventPointerId() {
		return pointerId;
	}

}

File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/NoCommand.kt
package com.example.try_gameengine.remotecontroller

import android.view.MotionEvent

/**
 * This no command is a default for remote controller.
 * @author irons
 // */
class NoCommand : Command {
    private var pointerId = -1

    override fun execute(): RemoteController.CommandType {
        return RemoteController.CommandType.None
    }

    override fun checkExecute(x: Float, y: Float, event: MotionEvent?): Boolean {
        return false
    }

    override fun setMotionEventPointerId(pointerId: Int) {
        this.pointerId = pointerId
    }

    override fun getMotionEventPointerId(): Int {
        return pointerId
    }
}
